assets/docs/emilutz/models/vgg19-block1-conv2-unpooling-decoder/1.md

```diff
@@ -0,0 +1,84 @@
+# Module emilutz/vgg19-block1-conv2-unpooling-decoder/1
```

Comment:
The handle in this line (and the location of the file in .../emilutz/models/vgg19-block1-conv2-unpooling-decoder/1.md) will make the model available at https://tfhub.dev/emilutz/vgg19-block1-conv2-unpooling-decoder/1 (so `tfhub.dev/HANDLE`). This doesn't match your code example i.e.
```python
decoder = hub.KerasLayer('https://tfhub.dev/vgg19-with-unpooling/vgg19-block1-conv2/decoder/1') # this model
```
I'd suggest that you update all the references to this model page to https://tfhub.dev/emilutz/vgg19-block1-conv2-unpooling-decoder/1.

Reply:
Thanks, I was actually confused about this aspect. I updated all references to all models.